{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/settings/base.py

```diff
@@ -209,110 +235,60 @@
-                'django.contrib.auth.context_processors.auth',
-                'django.template.context_processors.debug',
-                'django.template.context_processors.i18n',
-                'django.template.context_processors.media',
-                'django.template.context_processors.static',
-                'django.contrib.messages.context_processors.messages',
-                'django.template.context_processors.request',
-                'cms.context_processors.settings',
-                'cms.apps.pages.context_processors.pages',
-                'social_django.context_processors.backends',
-                'social_django.context_processors.login_redirect'
-            ]
-        }
-    },
-    {
-        'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [
-            os.path.join(SITE_ROOT, 'templates'),
-        ],
-        'APP_DIRS': True,
-        'OPTIONS': {
-            'context_processors': [
-                'django.contrib.auth.context_processors.auth',
-                'django.template.context_processors.debug',
-                'django.template.context_processors.i18n',
-                'django.template.context_processors.media',
-                'django.template.context_processors.static',
-                'django.contrib.messages.context_processors.messages',
-                'django.template.context_processors.request',
-                'cms.context_processors.settings',
-                'cms.apps.pages.context_processors.pages',
-                'social_django.context_processors.backends',
-                'social_django.context_processors.login_redirect'
-            ]
-        }
-    }
-]
 
+###
+#  _____  _     _          _                     _
+# |_   _|| |_  (_) _ _  __| |   _ __  __ _  _ _ | |_  _  _
+#   | |  | ' \ | || '_|/ _` |  | '_ \/ _` || '_||  _|| || |
+#   |_|  |_||_||_||_|  \__,_|  | .__/\__,_||_|   \__| \_, |
+#                              |_|                    |__/
+###
+AUTHENTICATION_BACKENDS = (
+    'social_core.backends.google.GooglePlusAuth',
+    'django.contrib.auth.backends.ModelBackend'
+)
+SUIT_CONFIG = {
+    'ADMIN_NAME': SITE_NAME,
+    'MENU_EXCLUDE': ['default'],
+}
+SOCIAL_AUTH_GOOGLE_PLUS_KEY = '{{cookiecutter.google_plus_key}}'
+SOCIAL_AUTH_GOOGLE_PLUS_SECRET = '{{cookiecutter.google_plus_secret}}'
+GOOGLE_ANALYTICS = '{{cookiecutter.google_analytics}}'
+ADMIN_ANALYTICS_ID = GOOGLE_ANALYTICS
+ADMIN_ANALYTICS_GOOGLE_API_KEY = '{{cookiecutter.google_analytics_key}}'
 
-# Namespace for cache keys, if using a process-shared cache.
+# You can get your Client ID & Secret here: https://creativesdk.adobe.com/myapps.html
+ADOBE_CREATIVE_SDK_ENABLED = {% if cookiecutter.adobe_creative_sdk_secret and cookiecutter.adobe_creative_sdk_id %}True{% else %}False{% endif %}
+ADOBE_CREATIVE_SDK_CLIENT_SECRET = '{{cookiecutter.adobe_creative_sdk_secret}}'
+ADOBE_CREATIVE_SDK_CLIENT_ID = '{{cookiecutter.adobe_creative_sdk_id}}'
```

Comment:
This stuff is unused now - probably remove this from here and from cookiecutter.json. (But also, let's see if we can find alternatives!)